schema/service.go

```diff
@@ -32,6 +32,8 @@ type MethodArgument struct {
 
 	InputArg  bool `json:"-"` // denormalize/back-reference
 	OutputArg bool `json:"-"` // denormalize/back-reference
+
+	TypeExtra `json:",omitempty"`
```

Comment:
This line embeds a new `.Meta` field in schema, which is not backward-compatible change.